script.js

```diff
@@ -0,0 +1,37 @@
+let books = [];
+const store = window.localStorage;
+const list = document.getElementById('booklist');
+const form = document.getElementById('form');
+
+function displayBooks() {
+  while (list.firstChild) {
+    list.removeChild(list.firstChild);
+  }
+  books = JSON.parse(localStorage.getItem('storeBook'));
+  books.forEach((book) => {
```

Comment:
**Kindly fix these errors that appear in console:**
![Screen Shot 2021-08-10 at 04 59 38](https://user-images.githubusercontent.com/22258274/128847739-840be9bf-c55b-4300-b000-5e187b4a651e.png)

**When a user clicks the "Add" button:**
1. A new book is added to the collection. 
2. The new book is displayed on the page.


**When a user clicks the "Remove" button:**
1. a. The right book is removed from the collection. 
2. b. The right book disappears from the page.

Reply:
> 
> 
> **Kindly fix these errors that appear in console:**
> ![Screen Shot 2021-08-10 at 04 59 38](https://user-images.githubusercontent.com/22258274/128847739-840be9bf-c55b-4300-b000-5e187b4a651e.png)
> 
> **When a user clicks the "Add" button:**
> 
>     1. A new book is added to the collection.
> 
>     2. The new book is displayed on the page.
> 
> 
> **When a user clicks the "Remove" button:**
> 
>     1. a. The right book is removed from the collection.
> 
>     2. b. The right book disappears from the page.

Hello Thanks for the review. I checked on my side it's working. I have no error in the console.
![image](https://user-images.githubusercontent.com/43698511/128850380-555d6fa3-9532-42aa-b5c8-713bb365935b.png)